### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (#q)

### ISSUE THREAD

**Issue by A11Ybot-OP** [bazelbuild/bazel#14476]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Bazel - a fast, scalable, multi-language and extensible build system" - Bazel](https://bazel.build/)
#### Element path
#q
#### Snippet
`<input type="search" name="q" id="q" class="form-control input-sm" placeholder="Search">`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.99 (foreground color: #555555, background color: #76d275, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62

**Comment by fweikert** [comment]
We've migrated to a new site at https://bazel.build that fixes several accessibility issues.
